Dalamud/Interface/InterfaceManager.cs

```diff
@@ -38,15 +36,18 @@ public class InterfaceManager : IDisposable
 
         private ISwapChainAddressResolver Address { get; }
 
+        private Dalamud dalamud;
         private RawDX11Scene scene;
 
         /// <summary>
         /// This event gets called by a plugin UiBuilder when read
         /// </summary>
         public event RawDX11Scene.BuildUIDelegate OnDraw;
 
-        public InterfaceManager(SigScanner scanner)
+        public InterfaceManager(Dalamud dalamud, SigScanner scanner)
```

Comment:
Why not pass just dalamud, and get the scanner from `dalamud.sigScanner`?

Is there any time where you might want to pass a different SigScanner?

Reply:
It's private and I try not to change interfaces unnecessarily.  This also follows the pattern of other objects initialized by dalamud.

Reply:
It does follow my pattern, but my pattern is just my pattern because I'm lazy :p
Either way it's fine really, if I ever do a general cleanup pass I'll change the DI setup into something more sensible